src/test/java/org/broadinstitute/hellbender/tools/walkers/vqsr/CNNScoreVariantsIntegrationTest.java

```diff
@@ -242,10 +246,24 @@ public void testInferenceArchitectureAndWeights2d() throws IOException {
                 .addArgument("tensor-type", tt.name())
                 .addArgument(StandardArgumentDefinitions.ADD_OUTPUT_VCF_COMMANDLINE, "false");
 
+        runCommandLine(argsBuilder);
+        assertInfoFieldsAreClose(tempVcf, expectedVcf, GATKVCFConstants.CNN_2D_KEY);
+    }
 
-        final IntegrationTestSpec spec = new IntegrationTestSpec(argsBuilder.toString(),
-                Arrays.asList(largeFileTestDir + "VQSR/expected/cnn_2d_chr20_subset_expected.vcf"));
-        spec.executeTest("testInference2d", this);
+    private void assertInfoFieldsAreClose(File vcf1, File vcf2, String infoKey){
```

Comment:
Rename all the args/variables in this method to have an `act`/`exp` or `actual`/`expected` prefix instead of `1`/`2` suffix.

Reply:
done